workspaces/topology/packages/app/src/App.tsx

```diff
@@ -117,8 +120,10 @@ export default app.createRoot(
   <>
     <AlertDisplay />
     <OAuthRequestDialog />
-    <AppRouter>
-      <Root>{routes}</Root>
-    </AppRouter>
+    <StyledEngineProvider injectFirst>
```

Comment:
Just a heads up that you should not need this feel free to look at what was done for the Backstage Demo site: https://github.com/backstage/demo/pull/664.